it/v2/endpoints/CreateUkPropertyPeriodSummaryControllerISpec.scala

```diff
@@ -503,26 +519,28 @@ class CreateUkPropertyPeriodSummaryControllerISpec extends V2IntegrationBaseSpec
           ("AA123456A", "XA***IS1", "2022-23", requestBodyJson, Status.BAD_REQUEST, BusinessIdFormatError),
           ("AA123456A", "XAIS12345678910", "2022-23", invalidToDateRequestJson, Status.BAD_REQUEST, ToDateFormatError),
           ("AA123456A", "XAIS12345678910", "2022-23", invalidFromDateRequestJson, Status.BAD_REQUEST, FromDateFormatError),
-          ("AA123456A", "XAIS12345678910", "2022-23", Json.parse(s"""{ "fromDate": "2020-04-06",
+          ("AA123456A",
+           "XAIS12345678910",
+           "2022-23",
+           Json.parse(s"""{ "fromDate": "2020-04-06",
                                                                     |  "toDate": "2019-04-06",
                                                                     |  "ukFhlProperty": {}
-                                                                    |  }""".stripMargin), BAD_REQUEST,
-            RuleIncorrectOrEmptyBodyError.copy(paths = Some(Seq("/ukFhlProperty")))),
+                                                                    |  }""".stripMargin),
+           BAD_REQUEST,
+           RuleIncorrectOrEmptyBodyError.copy(paths = Some(Seq("/ukFhlProperty")))),
           ("AA123456A", "XAIS12345678910", "2022-23", invalidValueRequestJson, Status.BAD_REQUEST, allInvalidValueRequestError),
           ("AA123456A", "XAIS12345678910", "2022-23", bothExpensesSuppliedRequestJson, Status.BAD_REQUEST, RuleBothExpensesSuppliedRequestError),
           ("AA123456A", "XAIS12345678910", "2022-23", toDateBeforeFromDateRequestJson, Status.BAD_REQUEST, RuleToDateBeforeFromDateError)
         )
         input.foreach(args => (validationErrorTest _).tupled(args))
       }
+
       "ifs service error" when {
         def serviceErrorTest(ifsStatus: Int, ifsCode: String, expectedStatus: Int, expectedBody: MtdError): Unit = {
-          s"ifs returns an $ifsCode error and status $ifsStatus" in new Test {
+          s"ifs returns an $ifsCode error and status $ifsStatus" in new TysIfsTest {
```

Comment:
Other repo uses NonTysTest for this. But I guess it doesnt really matter? 
Seems like it will still reach the same error mapping code